po/ja.po

```diff
@@ -0,0 +1,561 @@
+
+#: src/libgpaste/util/gpaste-util.c:410
+msgid "Do you really want to empty the history?"
+msgstr "本当に、履歴を空にしますか？"
+
+#: src/libgpaste/util/gpaste-util.c:709
+msgid "Could not create history dir"
+msgstr "履歴ディレクトリを作成できませんでした"
+
+#: src/client/gpaste-client.c:137 src/client/gpaste-client.c:425
+msgid "Couldn't spawn"
+msgstr "生成できませんでした"
+
+#: src/client/gpaste-client.c:149
+#, c-format
+msgid "Usage:\n"
+msgstr "使い方:\n"
+
+#. Translators: help for gpaste history
+#: src/client/gpaste-client.c:151
+msgid "print the history with indexes"
+msgstr "インデックス付きの履歴を表示する"
+
+#. Translators: help for gpaste history-size
+#: src/client/gpaste-client.c:153
+msgid "print the size of the history"
+msgstr "履歴の大きさを表示する"
+
+#. Translators: help for gpaste get-history
+#: src/client/gpaste-client.c:155
+msgid "get the name of the current history"
+msgstr "現在の履歴の名前を取得する"
+
+#. Translators: help for gpaste backup-history <name>
+#. Translators: help for gpaste switch-history <name>
+#. Translators: help for gpaste delete-history <name>
+#. Translators: help for gpaste add-password <name> <text>
+#. Translators: help for gpaste delete-passworf <name>
+#: src/client/gpaste-client.c:157 src/client/gpaste-client.c:159
+#: src/client/gpaste-client.c:161 src/client/gpaste-client.c:167
+#: src/client/gpaste-client.c:179 src/client/gpaste-client.c:183
+msgid "name"
+msgstr "名前"
+
+#: src/client/gpaste-client.c:157
+msgid "backup current history"
+msgstr "現在の履歴をバックアップする"
+
+#: src/client/gpaste-client.c:159
+msgid "switch to another history"
+msgstr "他の履歴に切り替える"
+
+#: src/client/gpaste-client.c:161
+msgid "delete a history"
+msgstr "履歴を削除する"
+
+#. Translators: help for gpaste list-histories
+#: src/client/gpaste-client.c:163
+msgid "list available histories"
+msgstr "利用可能な履歴を一覧表示する"
+
+#. Translators: help for gpaste add <text>
+#: src/client/gpaste-client.c:165
+msgid "text"
+msgstr "テキスト"
+
+#: src/client/gpaste-client.c:165
+msgid "set text to clipboard"
+msgstr "テキストをクリップボードに設定する"
+
+#: src/client/gpaste-client.c:167
+msgid "password"
+msgstr "パスワード"
+
+#: src/client/gpaste-client.c:167
+msgid "add the name - password couple to the clipboard"
+msgstr "クリップボードに名前・パスワードの一組を追加する"
+
+#. Translators: help for gpaste rename-password <old name> <new name>
+#: src/client/gpaste-client.c:169
+msgid "old name"
+msgstr "旧名"
+
+#: src/client/gpaste-client.c:169
+msgid "new name"
+msgstr "新しい名前"
+
+#: src/client/gpaste-client.c:169
+msgid "rename the password"
+msgstr "パスワードの名前を変更する"
+
+#. Translators: help for gpaste get <number>
+#. Translators: help for gpaste merge <number> … <number>
+#. Translators: help for gpaste set-password <number> <name>
+#. Translators: help for gpaste delete <number>
+#: src/client/gpaste-client.c:171 src/client/gpaste-client.c:177
+#: src/client/gpaste-client.c:179 src/client/gpaste-client.c:181
+msgid "number"
+msgstr "数字"
+
+#: src/client/gpaste-client.c:171
+msgid "get the <number>th item from the history"
+msgstr "履歴から<number>番目の項目を取得する"
```

Comment:
```suggestion
msgstr "履歴の <数字> 番目の項目を取得する"
```

You should translate this too :)

Reply:
Thanks. Was not sure if I should translate this or not, thought it was a program variable or parameter.